cli/src/checks.rs

```diff
@@ -31,7 +31,10 @@ pub fn check_overflow(cargo_toml_path: impl AsRef<Path>) -> Result<bool> {
 /// - `@coral-xyz/anchor` package version
 ///
 /// This function logs warnings in the case of a mismatch.
-pub fn check_anchor_version(cfg: &WithPath<Config>) -> Result<()> {
+pub fn check_anchor_version(
+    cfg: &WithPath<Config>,
+    package_manager: &PackageManager,
```

Comment:
`package_manager` parameter can be removed since we can get it from the first parameter (`cfg`).
```suggestion
```